tests/queries/0_stateless/02731_auto_convert_dictionary_layout_to_complex_by_complex_keys.sql

```diff
@@ -0,0 +1,29 @@
+DROP DICTIONARY IF EXISTS dict_flat_simple;
+DROP DICTIONARY IF EXISTS dict_hashed_simple;
+DROP DICTIONARY IF EXISTS dict_hashed_complex;
+DROP DICTIONARY IF EXISTS dict_hashed_simple_auto_convert;
+DROP TABLE IF EXISTS dict_data;
+
+CREATE TABLE dict_data (v0 UInt16, v1 UInt16, v2 UInt16, v3 UInt16, v4 UInt16) engine=Memory() AS SELECT number, number%65535, number%65535, number%6553, number%655355 FROM numbers(100);
```

Comment:
Not sure why we need such complex table and populate it with some data;
You can just create table `CREATE TABLE dict_data (v0 UInt16, v1 UInt16, v2 UInt16, v3 UInt16, v4 UInt16) engine=Memory()`.

Reply:
You are right, I have changed it